src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/Users/GetUserViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Users
+{
+    public sealed class GetUserViewModel
```

Comment:
As above, as we're not using this model in the context of a view I'm not sure it should have the `ViewModel` suffix (as it's potentially misleading).